### PR TITLE
Improve README

### PR DESCRIPTION
**Pull request by jakobj**
nice view: https://github.com/jakobj/python-gp/blob/maint/fix-readme/README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,22 +1,23 @@
...
+This library implements Cartesian Genetic Programming (e.g, Miller and Thomson, 2000; Miller, 2011) for symbolic regression in pure Python, targeting applications with expensive fitness evaluations. It provides Python data structures to represent and evolve two-dimensional directed graphs (genotype) that are translated into computational graphs (phenotype) implementing mathematical expressions. The computational graphs can be compiled as a Python functions, SymPy expressions (Meurer et al., 2017) or PyTorch modules (Paszke et al., 2017). The current implementation uses an evolutionary algorithm, specifically (mu + lambda) evolution strategies adapted from Deb et al. (2002), to evolve a population of symbolic expressions in order to optimize an objective function.
+
```
Comment (mschmidt87) [inline README.md]: I would rephrase the last sentence a bit. Rather than "current implementation", perhaps say "The library currently implements an evolutionary algorithm, specifically (mu + lambda) evolution strategies adapted from Deb et al. (2002), to evolve a population of symbolic expressions in order to optimize an objective function."

Comment (jakobj) [reply README.md]: ok

**Comment by jakobj** [comment]
thanks for the comment, implemented your suggestion @mschmidt87
